scarb/src/ops/compile.rs

```diff
@@ -219,7 +219,7 @@ fn compile_unit_inner(unit: CompilationUnit, ws: &Workspace<'_>) -> Result<()> {
             let ScarbDatabase {
                 mut db,
                 proc_macro_host,
-            } = build_scarb_root_database(&unit, ws)?;
+            } = build_scarb_root_database(&unit, ws, vec![])?;
```

Comment:
```suggestion
            } = build_scarb_root_database(&unit, ws, Default::default())?;
```

I know your code was shorter, but using `default` does better when refactoring